crates/typst-library/src/pdf/embed.rs

```diff
@@ -12,14 +12,14 @@ use crate::World;
 
 /// 出力されるPDFに埋め込まれるファイル。
 ///
-/// This can be used to distribute additional files that are related to the PDF
-/// within it. PDF readers will display the files in a file listing.
+/// この関数は、PDFに関連する追加のファイルをPDF内に埋め込んで配布するために使用できます。
+/// PDFリーダーはファイルリストにファイルを表示します。
 ///
-/// Some international standards use this mechanism to embed machine-readable
-/// data (e.g., ZUGFeRD/Factur-X for invoices) that mirrors the visual content
-/// of the PDF.
+/// 一部の国際規格では、
+/// この仕組みを使用して、
+/// PDFの視覚的な内容を反映した機械可読データ（例：請求書のZUGFeRD/Factur-X）を埋め込むことができます。
```

Comment:
特に助動詞は入っていないため、以下のような訳ではいかがでしょうか？

```suggestion
/// PDFの視覚的な内容を反映した機械可読データ（例：請求書のZUGFeRD/Factur-X）を埋め込んでいます。
```

Reply:
提案ありがとうございます、反映させていただきました。